### DISCUSSION TITLE
Is docker swarm strictly required?

### DISCUSSION THREAD

**Discussion by KROSF** [coollabsio/coolify#12] [Q&A] [answered]
i have a traefik instance so, i want to use that without enabling swarm mode

**Comment by andrasbacsai** [comment, marked as answer]
It's not an option because the applications are started in a stack, only available in swarm mode. Because at some point in the future, everything could be started with highly available replicas in swarm nodes.

I will check how much work necessary needed to have a non-swarm option as well.

**Reply by andrasbacsai** [reply]
With new v2, there is no need for swarm mode by default.
